backend/src/common/enums/plugin-names.enum.ts

```diff
@@ -0,0 +1,5 @@
+enum PluginNames {
+    AUTH_PLUGIN = 'authPlugin',
+}
+
+export { PluginNames };
```

Comment:
```suggestion
const PluginName = {
    AUTH: 'auth',
} as const;

export { PluginNames };
```

We don't use TS enums anymore, also name should be in singular form